src/event-helper.js

```diff
@@ -225,6 +234,14 @@ export function loadPromise(eleOrWindow) {
  *     case Windows.
  */
 function failedToLoad(eleOrWindow) {
+  // Mark the element as errored since some elements - like HTMLMediaElement
+  // using HTMLSourceElement - do not provide any synchronous way to verify if
+  // they already errored, even though the error event was already dispatched.
+  if (isHTMLMediaElement(eleOrWindow)) {
+    eleOrWindow[MEDIA_LOAD_FAILURE_SRC_PROPERTY] =
+      eleOrWindow.currentSrc || true;
```

Comment:
Is there a case where `currentSrc` is falsey?

Reply:
When the media has no source, like `<video></video>` or `<video><source></video>`, it gives an empty string